rasa/engine/graph.py

```diff
@@ -36,7 +36,9 @@ class SchemaNode:
         fn: The name of the function which should be called on the instantiated
             component when the graph is executed. The parameters from `needs` are
             filled from the parent nodes.
-        config: The configuration for this graph node.
+        config: The user's configuration for this graph node. This configuration
+            does not need to be specify all possible parameters; the default values
```

Comment:
A small typo here: `does not need to be specify` should be `does not need to specify`